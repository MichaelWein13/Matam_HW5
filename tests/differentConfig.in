The quick brown fox
   jumps over the lazy dog.

    Amidst the whispers of the wind,
        a single leaf falls,
            dancing gracefully in the autumn breeze.

Stars twinkle above, scattered like 
    diamonds across the velvet sky.

    Silence speaks louder than words,
        as the night slowly embraces the dawn.
