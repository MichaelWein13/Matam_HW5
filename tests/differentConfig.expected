Tis smylf zfdpi ubr
   ievjw lxoj ewc qmnb jfd.

    Alhbvi mkd spjqmctp dg rzf fflx,
        q gqjwch vbqg egacw,
            bxupjvd piqlsafokm zz zws tncehj ostdng.

Ssmff sczcujb rozsl, blmxqdtzf ptig 
    lnrhpwge qnhwtb rdf xjnxts iyf.

    Sycocqt tmbqyb ksjnbi qxhx ousep,
        qe bjo jjkpe puyoko bgshrlbk ded thno.
